src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpTraceExporter.cs

```diff
@@ -112,6 +99,18 @@ public override ExportResult Export(in Batch<Activity> activityBatch)
 
             request.AddBatch(this.ProcessResource, activityBatch);
 
+            if (this.options.TimeoutMilliseconds.HasValue)
+            {
+                if (this.options.TimeoutMilliseconds.Value <= 0)
+                {
+                    throw new ArgumentException("Timeout value provided is not a positive number.", nameof(this.options.TimeoutMilliseconds));
```

Comment:
please move the validation to ctor() itself. We cannot throw from Exporter, but is okay to thow from ctor.

Reply:
Moved the validation to ctor.